firmware/open_evse/open_evse.h

```diff
@@ -342,11 +342,6 @@ extern AutoCurrentCapacityController g_ACCController;
 #undef BTN_MENU
 #endif // RGBLCD || I2CLCD
 
-//If LCD and RTC is defined, un-define CLI so we can save ram space.
-#if defined(SERIALCLI) && defined(DELAYTIMER_MENU)
-#error INVALID CONFIG - CANNOT enable SERIALCLI with DELAYTIMER_MENU together - too big
-#endif
```

Comment:
I removed this check, as I didn't have any issue keeping both of these options on and building an image that wasn't oversized:

```
RAM:   [======    ]  57.9% (used 1186 bytes from 2048 bytes)
Flash: [========= ]  89.4% (used 29306 bytes from 32768 bytes)
```